src/main/java/frc/robot/commands/DefaultAimCommand.java

```diff
@@ -31,18 +31,19 @@ else if (Input.getY()) {
       mAimingSubsystem.setDesiredSetpoint(AimState.SPEAKER);
     }
     else if (Input.getDPad() == 0.0) {
-      mAimingSubsystem.setDesiredSetpoint(AimState.CLIMB);    }
+      mAimingSubsystem.setDesiredSetpoint(AimState.CLIMB);
+    }
 
 
     if (Math.abs(Input.getLeftStickY()) > 0) {
       //convert between input to increment
-      double increment = Input.getLeftStickY() * AimingConstants.MAX_ELEVATOR_TELEOP_INCREMENT;
+      double increment = Input.getLeftStickY() * AimingConstants.MAX_WRIST_TELEOP_INCREMENT;
       mAimingSubsystem.changeDesiredWristRotation(increment);
     }
     
     if (Math.abs(Input.getRightStickY()) > 0) {
       //convert between input to increment
-      double increment = Input.getRightStickY() * AimingConstants.MAX_WRIST_TELEOP_INCREMENT;
+      double increment = Input.mXboxController.getRightY() * AimingConstants.MAX_ELEVATOR_TELEOP_INCREMENT;
```

Comment:
Can you make the getRightY a method in Input rather than accessing mXboxController directly?

Reply:
I realized that we already had a method for this, but we multiplied by -1 because it was returning inverted values.
So, I just called the existing method & multiplied the returned value by -1.